### ISSUE TITLE
Does 3.0 not include the I2C optimizations?

### ISSUE THREAD

**Issue by qwertychouskie** [Eeshwar-Krishnan/PhotonFTC#13]
Looks like https://github.com/Eeshwar-Krishnan/PhotonFTC/commit/e63028a96b865c7b4676c61336d638719eed957a removed the I2C optimizations?  Or am I missing something?

**Comment by Eeshwar-Krishnan** [comment]
Yes, due to sporadic crashing issues on the BHI260AP IMU enabled hubs, they were removed for release. Further testing is being done in order to bring them back for the next PhotonCore version.

**Comment by qwertychouskie** [comment]
Is there a way to enable them anyways?  We use older hubs with the original IMU, which from my understanding is not affected by the issue.

**Comment by IronReignRobotics** [comment]
yes please - we also use BNO055 hubs and would love us some i2c performance
